building/build-debs/homeworld-admin-tools/src/command.py

```diff
@@ -5,8 +5,11 @@ class CommandFailedException(Exception):
     pass
 
 
-def fail(message: str) -> None:
-    raise CommandFailedException(str(message))
+def fail(message: str, hint: str = None) -> None:
+    formatted_message = '\033[91m' + str(message)
+    if hint is not None:
+        formatted_message += '\n\033[93m' + str(hint)
+    raise CommandFailedException(formatted_message)
```

Comment:
Besides the fact that this is being fixed elsewhere (note that #160 is claimed), it probably doesn't make sense to add color at this point: it probably makes more sense to add color when CommandFailedException is caught on the other end.

I do like your idea of having a "hint" section, however, so I recommend you expand CommandFailedException to have that as an additional attribute and print it out when CommandFailedException is caught.